src/shared/components/VideoPlayer/VideoPlayer.style.tsx

```diff
@@ -31,7 +31,10 @@ const backgroundContainerCss = css`
 
 export const Container = styled.div<ContainerProps>`
   position: relative;
-
+  height: 30vh;
```

Comment:
We shouldn't duplicate these values across two components (this and `PlayerPlaceholder`). I think we can move these styles to `PlayerContainer` and then set `height: 100%` for both this and `PlayerPlaceholder`?

Reply:
Done